scl/elasticsearch/plugin.conf

```diff
@@ -68,6 +68,14 @@ block destination elasticsearch2(
   concurrent_requests("1")
   skip_cluster_health_check("")
   cluster_url("")
+  java_keystore_filepath("")
+  java_keystore_password("")
+  java_truststore_filepath("")
+  java_truststore_password("")
+  java_ssl_insecure("false")
```

Comment:
java_ssl_insecure:
why we need this option? I mean if truststore is not given, than we know that this will be insecure. So instead of a different option (that can be confusing for example insecre(true) but the truststore is given) I would use the truststore value to decide it